.github/workflows/contrib-tests.yml

```diff
@@ -256,6 +256,45 @@ jobs:
           file: ./coverage.xml
           flags: unittests
 
+  GeminiTest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-2019]
+        python-version: ["3.12"]
```

Comment:
I would expect more problems with lower versions than with 3.12. Wouldn't it be better to run CI with 3.8?